m2m-transfers/m2m-transfers-json-contract.md

```diff
@@ -52,21 +52,20 @@ list of institution and member limits, eligible shares, list of scheduled transf
 		],
 
 		"scheduledTransfers": [{
-				"trnsferLoc": "215",
+				"transferLoc": "215",
 				"sourceAccount": "1234567890S0020",
 				"accountName": "CHECKING",
 				"transferAmt": "120.00",
 				"recipientName": "Emily Fitch",
 				"recipientMemberId": "9876543210",
-				"recipientAccount": "[9876543210L0001,savings,checking]",
+				"recipientAccount": "9876543210L0001", // could be specific account, or "savings" or "checking"?
 				"recipientNickname": "Emmy",
-				"startDate":"07/31/21",
-				"endDate":"12/31/2021",
-				"transferFrequency": "weekly",
-				"day1": "",
-				"day2": "",
-				"saveRecipient": false,
-				"memo": "Internal Note created by member"
+				"startDate": "07/31/21",
+				"endDate": "12/31/2021", // unused
```

Comment:
date formats should be consistent. 2 or 4-digit year.

Reply:
I vote 4 digit